plugins/vivisect-broker/broker/vivserver.proto

```diff
@@ -8,12 +8,19 @@ message RunInstructionsRequest {
     int32 numInstructions = 1;
 }
 
+message MemoryAccess {
+    uint32 index = 1;
```

Comment:
What is this `index` field for?

Reply:
It is the index of the memory access instruction in the enclosing TranslationBlock (in the Qemu Broker). In this case, it may represent the instruction index in the enclosing BasicBlock. Not sure yet if its entirely necessary, but I'm trying to reduce parity

Reply:
On second thoughts, I’ll reconfirm if its necessary to include this field.

Reply:
I think its worthwhile to keep the instruction `index` field around. The `fetchRegion` interface of the Broker requests `MCInsts` whereas the memory access metadata is attached through the `MDExchanger`. The `index` field is necessary to maintain a mapping between them.

Reply:
Right I see that it's necessary to pass in the index of the instruction corresponding to the memory access, but it seems unnecessary to explicitly put it in given that the interface already maps each instruction to an optional memory access and you can get instruction indices [like this](https://github.com/immunant/LLVM-MCA-Daemon/blob/main/plugins/vivisect-broker/broker/Broker.cpp#L68).